openedx/core/lib/new_relic_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Contains utilities for working with New Relic.
```

Comment:
Nit: I don't love the idea of having a file dedicated to a particular product. Maybe rename it to analytics_utils.py?